docker/agora/services/data.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  agora-data:
+    image: ghcr.io/sage-bionetworks/agora-data:${AGORA_VERSION:-local}
+    container_name: agora-data
+    restart: no
+    env_file:
+      - ../../../apps/agora/data/.env
+    environment:
+      - PYTHONUNBUFFERED=1
```

Comment:
Ensure that all print statements are logged in the detached `agora-data` container, see: https://stackoverflow.com/a/59582063